ganga/GangaCore/Utility/Monitoring.py

```diff
@@ -0,0 +1,54 @@
+from GangaCore.Utility.Config import getConfig
```

Comment:
Is this a new file? Perhaps a better name would be `Profiling`? `Monitoring` makes me think of other things, like the job monitoring.

Reply:
Yes, `Profiling` would be a better name.